workflow-templates/knative-go-build.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 The Knative Authors.
```

Comment:
2022?

Reply:
oh whoops - I copied and pasted - let me update

Reply:
Actually - can we skip this change - otherwise it'll make a whole bunch of necessary PRs just updating this - and updating the date doesn't really buy us anything wrt. to copyright

Reply:
sounds good, not a big deal